sql_normalization/sql_normalization.md

```diff
@@ -0,0 +1,370 @@
+<!--
+author:   Joy Payton
+email:    paytonk@chop.edu
+version:  1.0.0
+module_template_version: 3.0.0
+language: en
+narrator: US English Female
+title: SQL Normalization
+comment: Learn about the concept of normalization and why it's important for SQL.
```

Comment:
Bigger question here... yes this is how SQL is organized, but it is also just pure information theory.

I can see this module being assigned after the "tidy data" module, and it really is more about data management than about learning SQL
```suggestion
comment: Learn about the concept of normalization and why it's important for organizing complicated data.
```
```suggestion
comment: Learn about the concept of normalization and why it's important for SQL.
```

Reply:
ooooh you are so right